app/views/deploys/recent.html.erb

```diff
@@ -1,6 +1,7 @@
 <h1><%= title %></h1>
 
 <div class="timeline" ng-controller="<%= controller %>">
+  <input type=hidden  id=default_time_format value="<%= current_user.time_format %>">
```

Comment:
something about the interplay between the angular filter etc is made sad by this.